applications/CoSimulationApplication/python_scripts/convergence_accelerators/convergence_accelerator_wrapper.py

```diff
@@ -20,7 +20,20 @@ def __init__(self,
                  settings: KratosMultiphysics.Parameters,
                  interface_data_dict: "dict[str,CouplingInterfaceData]",
                  parent_coupled_solver_data_communicator: KratosMultiphysics.DataCommunicator):
-        self.interface_data = interface_data_dict[settings["data_name"].GetString()]
+        if settings.Has("solver_sequence"):
+            self.interface_data_dict = interface_data_dict
+            self.solver_names_sequence = []
+            self.coupl_data_names = {}
+            self.input_data = {}
+            for data_name in self.interface_data_dict:
+                self.solver_names_sequence.append(self.interface_data_dict[data_name].solver_name)
+            self.isBlockAccelerator = True
```

Comment:
```suggestion
            self.is_block_accelerator = True
```

Reply:
also please initialize this to `False`, or `None`, depending on what you think is more appropriate